helm/loki/templates/_helpers.tpl

```diff
@@ -286,3 +287,153 @@ http {
   }
 }
 {{- end }}
+
+{{/*
+/!\ Giantswarm override to work around a bug in azure where `use_federated_token` fails.
+/!\ Added on chart 4.10.0 / Loki 2.7.2
+Generated storage config for loki common config
+*/}}
```

Comment:
I see 2 long section here defining storage for aws, azure, gcs, minio, filesystem. How is this only a workaround for azure ? I am confused.

Reply:
We're overriding the original _helpers `loki.commonStorageConfig` definition.
So we retrieved the loki chart one (copy/paste) and updated (removed) the `use_federated_token` definition from the Azure part.
But we had to take the whole "function".

Reply:
Then I would suggest to mention what you just said, something like: "This was copied from upstream and ˋuse_federated_token` was removed for azure" also link where this was taken from.

Reply:
ok, I added this comment and added a "remove after" note.